jobs/aws/eks-anywhere-packages/eks-anywhere-packages-presubmits.yaml

```diff
@@ -0,0 +1,45 @@
+# Copyright Amazon.com Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+presubmits:
+  aws/eks-anywhere-packages:
+    - name: eks-anywhere-packages-presubmit
+      always_run: false
+      run_if_changed: ".*"
+      max_concurrency: 10
+      cluster: "prow-presubmits-cluster"
+      skip_report: false
+      decoration_config:
+        gcs_configuration:
+          bucket: s3://prowpresubmitsdataclusterstack-prowbucket7c73355c-vfwwxd2eb4gp
```

Comment:
@abhay-krishna is this the correct bucket to use?

Reply:
Yes that's the right one